### PR TITLE
TE-2864: Revised content

### PR DESCRIPTION
**Pull request by maro-vm**

### CODE REVIEW

**Comment by maro-vm** [comment]
image

image

image

File: README.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
+<!-- START_METADATA
 ---
-title: Introduction
```
Comment (OFollan) [inline README.md:2]: uuh, don't we need this one? Does it magically default to the #Introduction on line 10?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+END_METADATA -->
+
+# Introduction
 
-# Webhooks API [BETA]
+💥 Work in progress 💥
 
-An API to register one or more callback urls per named event per sales unit.
+You can **subscribe** to published **event types** by registering webhooks and
+will then be **notified** when any matching events are published for payments
```
Comment (OFollan) [inline README.md]:
```suggestion
will then be **notified** when any matching events are published
```
I guess we can leave `payments` out of it, to make it generic enough for e.g. Login events

File: README.md [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
+that match the given sales unit id.
```
Comment (OFollan) [inline README.md]:
```suggestion
that match the given Sales unit.
```
Hm - what is the public term we use for this again 🙈 ? Halp @khenrichsen

File: faq.md [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
 sidebar_position: 3
+pagination_next: null
+pagination_prev: null
 ---
+END_METADATA -->
 
-- [How to replace a webhook](#how-to-replace-a-webhook)
+# Frequently Asked Questions
 
+Here are the Report API Frequently Asked Questions (FAQ).
```
Comment (OFollan) [inline faq.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (rebekaburnett) [reply README.md:2]: I'd like us to use this metadata:
```
---
title: Introduction to the Webhooks API
sidebar_label: Introduction
sidebar_position: 1
hide_table_of_contents: true
pagination_prev: null
pagination_next: null
---
```

File: event-types.md [hunk 5]
```diff
@@ -1,12 +1,19 @@
+<!-- START_METADATA
 ---
-title: Events
 sidebar_position: 2
+pagination_next: null
+pagination_prev: null
 ---
+END_METADATA -->
```
Comment (rebekaburnett) [inline event-types.md]: I'd like to use this, or similar:
```
title: Webhooks API Events
sidebar_label: Events
sidebar_position: 2
pagination_next: null
pagination_prev: null
```

File: faq.md [hunk 6]
```diff
@@ -1,19 +1,24 @@
+<!-- START_METADATA
 ---
-title: FAQ
+sidebar_label: FAQ
 sidebar_position: 3
+pagination_next: null
+pagination_prev: null
 ---
```
Comment (rebekaburnett) [inline faq.md:8]: I'd like to use this, or similar:

```
title: Webhooks API Frequently Asked Questions
sidebar_label: FAQ
sidebar_position: 3
pagination_next: null
pagination_prev: null
```

File: README.md (hunk 3 above)
Comment (rebekaburnett) [reply README.md]: We usually say "sales unit"

File: README.md (hunk 1 above)
Comment (maro-vm) [reply README.md:2]: It does magically default to #Introduction. By default the title tag is actually ignored if there is a top level title in markdown (#). 

@rebekaburnett, I still think it's a bit weird to include the context in the title AND have the context clearly defined in the bread crumb just above. 

No matter. I'll update! =)

File: README.md (hunk 2 above)
Comment (maro-vm) [reply README.md]: Sure. Thanks. =)

File: event-types.md (hunk 5 above)
Comment (maro-vm) [reply event-types.md]: Sure. Will update. I assume this leads to replacing event type with event too?

**Review by rebekaburnett — APPROVED** [review]
Looks good!!

**Pull request merged**